<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerControlWidget</class>
 <widget class="QWidget" name="PlayerControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_passed_time">
         <property name="text">
          <string>Passed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_timeline">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_remained_time">
         <property name="text">
          <string>Remained</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_playing_type">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icones.qrc">
           <normaloff>:/icone/repeatone.png</normaloff>:/icone/repeatone.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_previous">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icones.qrc">
           <normaloff>:/icone/previous.png</normaloff>:/icone/previous.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_play">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icones.qrc">
           <normaloff>:/icone/play.png</normaloff>:/icone/play.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_next">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icones.qrc">
           <normaloff>:/icone/next.png</normaloff>:/icone/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_mute">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icones.qrc">
           <normaloff>:/icone/volume.png</normaloff>:/icone/volume.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_volume">
           <property name="text">
            <string>Volume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_volume">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/icones.qrc"/>
 </resources>
 <connections/>
</ui>
